probtorch/objectives/marginal.py

```diff
@@ -80,7 +80,7 @@ def elbo(q, p, sample_dim=None, batch_dim=None, alpha=0.1, beta=(1.0, 1.0, 1.0,
 
 
 def kl(q, p, sample_dim=None, batch_dim=None, log_weights=None, beta=(1.0, 1.0, 1.0, 1.0, 1.0),
-         size_average=True, reduce=True, bias=None):
+       size_average=True, reduce=True, bias=None):
```

Comment:
This appears to just be a whitespace edit.  I'm fine with it going in if JW is.

Reply:
only review-worthy change is the `sample_dim` thing, rest is pep8 etc.- don't waste your time on those...